fabfile/fab_assert.py

```diff
@@ -104,10 +104,8 @@ def coverage() -> None:
     if os.path.isfile(COVERAGE_REPORT_FILE):
         os.remove(COVERAGE_REPORT_FILE)
 
-    initialize_db()
-    local("coverage run --source='.' manage.py {0}".format(get_test_command(
-        parallel=False, fake_migrations=False
-    )))
+    # initialize_db()
```

Comment:
Why?

Reply:
The whole thing will be using in memory SQLite database.
The initialize_db() method is used to initialize a PostrgeSQL database.